### ISSUE TITLE
Train/Test evaluation

### ISSUE THREAD

**Issue by avieini** [CamDavidsonPilon/lifelines#388]
Hi,
Thank's for this awsome package.
I wondered if I can use this framework within Machine Learning context.
My plan is to divide the dataset to train/test sets and run a model with different parameters each time. 
than, exmaining it on the test set. 
my question is how can I examine the model on the test set? how can I run on the test set the check?
I know there is the cv option but I want somtething else..
please help!
thanks

**Comment by CamDavidsonPilon** [comment]
> I know there is the cv option but I want somtething else..

What else are you thinking, @avieini?

**Comment by avieini** [comment]
something like grid search (like in Scikit-learn) for the "alpha" or stepwise model selection.
do you think there is an option to to do this nice way?

**Comment by CamDavidsonPilon** [comment]
You probably want to try the library https://pypi.python.org/pypi/scikit-survival if that is your primary objective
